tests/utils/logger.py

```diff
@@ -0,0 +1,29 @@
+import os
+import logging
+
+from root import ROOT_DIR
+
+
+class Logger:
```

Comment:
Why do we need a custom Logger? why not just use the standard python one?

Reply:
While not a necessity, having a centralized class for the logging to handle level, path & formatting will make it easy for any global logging config changes needed.   Should we want a different log level  or location  for a specific test, the standard one could be defined and used as well.